collectors/metadata/schemas/definitions.json

```diff
@@ -22,6 +22,11 @@
         "icon_filename": {
           "type": "string",
           "description": "The filename of the integration's icon, as sourced from https://github.com/netdata/website/tree/master/themes/tailwind/static/img."
+        },
+        "deploy-on": {
+          "type": "string",
+          "description": "This is a tag that indicates where this integration can be deployed. Add it only if this integration is inside categories of the `Deploy` master category. That way the automation will know to add the proper command.",
+          "oneOf": ["linux","macos","docker","k8s"]
```

Comment:
What does it mean "this integration can be deployed"? Can you provide examples of such integrations (assuming they are not data collection integrations)?